doc/sphinx/source/ops/pad.rst

```diff
@@ -28,15 +28,15 @@ Inputs
 Attributes
 ----------
 
-+-----------------------+-------------------------------------------------------+
-| Name                  | Description                                           |
-+=======================+=======================================================+
-| ``padding_below``     | Shape of padding added before ``arg``                 |
-+-----------------------+-------------------------------------------------------+
-| ``padding_above``     | Shape of padding added after ``arg``                  |
-+-----------------------+-------------------------------------------------------+
-| ``padding_interior``  | Shape of padding inserted between elements of ``arg`` |
-+-----------------------+-------------------------------------------------------+
++-----------------------+---------------------------------------------------------------------+
+| Name                  | Description                                                         |
```

Comment:
The `pad_mode` attribute should be documented as well, and the mention of `padding_interior` removed. (We can punt to a later PR, but let's open a JIRA if we do.)

Reply:
I modified documentation tables but example must be modified.
Should `get_padding_interior()` be removed from `src/ngraph/op/pad.cpp`?

Reply:
`get_padding_interior` is deprecated but cannot be removed until all backends have stopped using it. (Currently it just returns a vector of all ones.) Not necessary to remove it in this PR, but nice to have if we happen to get there.